test/gurka/test_ramps_tc.cc

```diff
@@ -279,3 +279,148 @@ TEST(RampsTCs, test_tc_infer) {
       result, maneuver_index, "Turn left onto KJ.", "Turn left onto KJ.",
       "Turn left onto KJ. Then You will arrive at your destination.", "Continue for 50 meters.");
 }
+
+namespace {
+void check_edge_classification(baldr::GraphReader& graph_reader,
+                               const gurka::nodelayout& nodes,
+                               const std::string& b,
+                               const std::string& e,
+                               baldr::RoadClass rc) {
+  const auto edge = std::get<1>(gurka::findEdgeByNodes(graph_reader, nodes, b, e));
+  EXPECT_EQ(edge->classification(), rc);
+}
+} // namespace
+
+TEST(LinkReclassification, test_use_refs) {
```

Comment:
check that `ref` and `destination:ref` tags are used correctly to detect destination roads. We should use destination road class instead of intermediate roads classes in order to determine final link classification